pkg/cluster/cluster.go

```diff
@@ -362,24 +362,23 @@ func (c *Cluster) delete() {
 
 	pods, err := c.KubeCli.Pods(c.Namespace).List(option)
 	if err != nil {
-		panic(err)
+		c.logger.Errorf("fail to list pods: %v", err)
```

Comment:
if this fails, we cannot really remove any pods. we need to explicitly tell that to users. also we need to add delete prefix so that users know this happens in delete call.